api.md

```diff
@@ -170,6 +172,75 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
+3. Loading the TimescaleDB extension on the new database
+
+#### Errors
+
+An error will be given if:
+* an attempt is made to run the function inside a transaction _or_
+* the remote database already exists on the data node and
+  `if_not_exists` is not set.
+
+#### Required Arguments [](add_data_node-required-arguments)
+
+| Name        | Description                         |
+| ----------- | -----------                         |
+| `node_name` | Name for the data node.             |
+
+#### Optional Arguments [](add_data_node-optional-arguments)
+
+| Name                 | Description                                           |
+|----------------------|-------------------------------------------------------|
+| `host`               | Host name for the remote data node. The default is `'localhost'`. |
+| `port`               | Port to use on the remote data node. The default is the PostgreSQL port used by the access node on which the function is executed. |
+| `database`           | Database name where remote hypertables will be created. The default is the current database name. |
+| `password`           | Password to be used on the remote data node. The default is to not use a password. |
+| `if_not_exists`      | Do not fail if the data node already exists. The data node will not be overwritten, but the command will not generate an error and instead generate a notice. |
+| `bootstrap_database` | Database to use when bootstrapping as described above. The default is `'postgres'`. |
+| `bootstrap_user`     | User to be used for bootstrapping. This user needs to have `SUPERUSER` permissions on the remote database server, unless the remote database already exists with the timescaledb extension. The default is the current user. |
+| `bootstrap_password` | Password to be used when bootstrapping. The default is the password provided in `password`. |
+
+#### Returns [](add_data_node-returns)
+
+| Column              | Description                                       |
+|---------------------|---------------------------------------------------|
+| `node_name`         | Local name to use for the data node               |
+| `host`              | Host name for the remote data node                |
+| `port`              | Port for the remote data node                     |
+| `database`          | Name of database used on the remote data node     |
+| `node_created`      | Was the node created                              |
+| `database_created`  | Was the database created on the remote data node  |
+| `extension_created` | Was the extension created on the remote data node |
+
+#### Sample usage [](add_data_node-examples)
+
+Let's assume that you have an existing hypertable `conditions` and
+want to use `time` as the time partitioning column and `location` as
+the space partitioning column. You also want to distribute the chunks
+of the hypertable on two data nodes `dn1.example.com` and
+`dn2.example.com`:
```

Comment:
Isn't it good to mention that the current user should be a superuser to succeed? Otherwise I am confused about superuser requirement for bootstrap user.

Reply:
Yeah, I added a note and provided a more fleshed out example.